chart/templates/scheduler/scheduler-vpa.yaml

```diff
@@ -0,0 +1,61 @@
+{{/*
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+*/}}
+
+################################
+## Airflow Scheduler VPA
+#################################
+{{- if semverCompare "<3.0.0" .Values.airflowVersion }}
```

Comment:
Do I see it right that you want to enable VPA only if Airflow-version is <3.0.0?

If contributed it should also support all current versions, might exclude (very) old versions.